### DISCUSSION TITLE
How do we parameterize any object other than linked service in the json config file

### DISCUSSION THREAD

**Discussion by nikhil-neu** [Azure-Player/azure.datafactory.tools#163] [Q&A] [answered]
I have parameterized my linked services objects using the following syntax
`{
  "akv_ls_amnoneshd": [
    {
      "name": "$.properties.typeProperties.baseUrl",
      "value": "$($Env:kvURI)",
      "action": "update"
    }
  ],
  "sql_ls_amie_amhdb": [
    {
      "name": "$.properties.connectVia.referenceName",
      "value": "$($Env:ADFIntegrationRuntime)",
      "action": "update"
    }
  ], ` 
 
Here the top level element is the name of the linked service itself .

How do i parameterize other objects like triggers for example .

Does json config by default look for linked services ?

**Comment by NowinskiK** [comment, marked as answer]
There is an example of JSON config file here: https://github.com/SQLPlayer/azure.datafactory.tools#json-format-of-config-file
Just add another object to the array.
JSON config like lookups for objects by name only, so the names have to be unique across the entire ADF.
